sdk/python/featurestore_sample/project/fraud_model/pipelines/training_pipeline.yaml

```diff
@@ -80,7 +80,7 @@ jobs:
 
   register_model_step:
     type: command
-    compute: azureml:cpu-cluster
+    compute: azureml:cpu-cluster-fs
```

Comment:
Do we need to update the inference pipeline as well?

Reply:
inference pipeline was already updated to use -fs.